docs/source/write_first_app.rst

```diff
@@ -1,1014 +1,572 @@
-
-In the sample application code below, you will see that after getting reference to the
-common connection profile path, making sure the connection profile exists, and specifying where to create the wallet,
-``enrollAdmin()`` is executed and the admin credentials are generated from the Certificate Authority.
-
-.. code:: bash
-
-  async function main() {
-    try {
-      // build an in memory object with the network configuration (also known as a connection profile)
-      const ccp = buildCCP();
+The ``src`` directory contains the client application source code. The JavaScript output generated from this source
+code during the install process is located in the ``dist`` directory, and can be ignored.
 
-      // build an instance of the fabric ca services client based on
-      // the information in the network configuration
-      const caClient = buildCAClient(FabricCAServices, ccp);
 
-      // setup the wallet to hold the credentials of the application user
-      const wallet = await buildWallet(Wallets, walletPath);
+Run the sample application
+==========================
+When we started the Fabric test network earlier in this tutorial, several identities were created using the Certificate
+Authorities. These include a user identity for each of the organizations. The application will use the credentials
+of one of these user identities to transact with the blockchain network.
 
-      // in a real application this would be done on an administrative flow, and only once
-      await enrollAdmin(caClient, wallet);
-
-This command stores the CA administrator's credentials in the ``wallet`` directory.
-You can find administrator's certificate and private key in the ``wallet/admin.id``
-file.
-
-.. note:: If you decide to start over by taking down the network and bringing it back up again, you will
-          have to delete the ``wallet`` folder and its identities prior to re-running the javascript application
-          or you will get an error. This happens because the Certificate Authority and its database are taken down
-          when the test-network is taken down but the original wallet still remains in the application-javascript directory
-          so it must be deleted.  When you re-run the sample javascript application, a new wallet and credentials will
-          be generated.
+Let's run the application and then step through each of the interactions with the smart contract functions. From the
+``asset-transfer-basic/application-gateway-typescript`` directory, run the following command:
 
+.. code-block:: bash
 
-If you scroll back up to the beginning of the output in your terminal, it should be similar to below:
+  npm start
 
-.. code:: bash
 
-  Wallet path: /Users/<your_username>/fabric-samples/asset-transfer-basic/application-javascript/wallet
-  Successfully enrolled admin user and imported it into the wallet
+First, establish a gRPC connection to the Gateway
```

Comment:
As before, discussing peers was confusing so it's avoided.

Reply:
@joshhus What is the concern? I think it is more confusing to avoid using the term peer. For the peer topic, the resolution was to use language like "Gateway service running on a peer". If we don't do some kind of clarification like this, people will get the impression that the Gateway is its own runtime component. Or worse, might think we're talking about the Gateway within the client SDK since that is what it used to be called.